lib/ecto/adapter/migration.ex

```diff
@@ -54,7 +54,7 @@ defmodule Ecto.Adapter.Migration do
       to finish, `:infinity` will wait indefinitely (default: 5000);
     * `:log` - When false, does not log begin/commit/rollback queries
   """
-  @callback execute_ddl(adapter_meta, command, options :: Keyword.t()) :: {:ok, list()}
+  @callback execute_ddl(adapter_meta, command, options :: Keyword.t()) :: [{Logger.level, Logger.message, Logger.metadata}]
```

Comment:
Sorry, I don't think it was precise though, it should still be `{:ok, [{...}]}`. It is important to return `{:ok, something}` in case we want to return `:error` in the future.